daemon/containerfs_linux.go

```diff
@@ -17,7 +17,7 @@ import (
 
 	"github.com/docker/docker/daemon/container"
 	"github.com/docker/docker/daemon/internal/mounttree"
-	"github.com/docker/docker/internal/unshare"
+	"github.com/docker/docker/daemon/internal/unshare"
```

Comment:
Not for this PR; considering if this package could be moved to a module in `moby/sys`; it looks very self-contained and stable, and we currently maintain a fork of the code in `moby/go-archive/chrootarchive`.